services/QuillLMS/app/services/classlink_integration/constants.rb

```diff
@@ -3,6 +3,6 @@
 module ClasslinkIntegration
   module Constants
     APP_SETTING = 'classlink_integration_enabled'
-    DISTRICT_TENANT_ID_ALLOWLIST = 'CLASSLINK_DISTRICT_TENANT_ID_ALLOWLIST'
+    SMEAR_RATE_IN_SECONDS = 'CLASSLINK_SMEAR_RATE_IN_SECONDS'
```

Comment:
Just out of curiosity, what's a smear rate? Google turns up information about cancer screenings.

Reply:
@emilia-friedberg   it's jargon for "programatically scheduling async jobs in the future to avoid performance bottlenecks" 

See also: https://github.com/empirical-org/Empirical-Core/blob/850a23bd160f89f5a924f3fccd0293c3ae5909ec/services/QuillLMS/app/services/learn_worlds_integration/sync_orchestrator.rb#L5

If it were up to me I would have picked a different word